packages/editor/src/browser/editor.ts

```diff
@@ -168,6 +168,9 @@ export interface TextEditor extends Disposable, TextEditorSelection, Navigatable
 
     readonly onMouseDown: Event<EditorMouseEvent>;
 
+    readonly onScrollChanged: Event<void>;
+    getVisibleLineRange(): Range | undefined;
```

Comment:
please rename to `getVisibleRanges` then

Reply:
@AlexTugarev Done